app/modules/advert/endpoints_advert.py

```diff
@@ -268,19 +268,21 @@ async def create_advert(
     try:
         now = datetime.now(UTC)
         message = Message(
-            context=f"advert-{result.id}",
+            context=f"new-advert-{id}",
```

Comment:
I would be in favor of always prefixing the context by the module name to avoid conflicts between modules.
```suggestion
            context=f"advert-new-{id}",
```